packages/lesmis/src/components/DashboardExportModal/components/A4Page.js

```diff
@@ -2,32 +2,22 @@ import React from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { FlexColumn } from '@tupaia/ui-components';
-import { useIsFetching } from 'react-query';
 
 import Header from './Header';
 
 const A4Container = styled.div`
-  width: 1300px;
+  width: 1192px;
```

Comment:
I don't think we need to specify a fixed width for printing, it will figure it out for us no? I just went to print this page in my browser and it fits nicely on A4, even though its like 2000px wide.

Actually you can delete this whole component no?

Reply:
Discussed. Need fixed width because of how our charts render.
Need component because of this.